pkg/generic/thrift/json.go

```diff
@@ -141,10 +247,18 @@ func (m *ReadJSON) Read(ctx context.Context, method string, in thrift.TProtocol)
 	}
 
 	// resp is map
-	respNode, err := jsoniter.Marshal(resp)
+	respNode, err := customJson.Marshal(resp)
 	if err != nil {
 		return nil, perrors.NewProtocolErrorWithType(perrors.InvalidData, fmt.Sprintf("response marshal failed. err:%#v", err))
 	}
 
 	return string(respNode), nil
 }
+
+// removePrefixAndSuffix removes json []byte from prefix `{"":` and suffix `}`
+func removePrefixAndSuffix(buf []byte) []byte {
```

Comment:
@felix021 I'm sure that the prefix is always `{"":` because dynamicgo writes `{` first and secondary writes `"response_field_alias":`, and the alias name of response is always empty.